.github/workflows/integration.yml

```diff
@@ -38,7 +38,13 @@ jobs:
 
       - name: Bring the stack up
         working-directory: ./ops
-        run: docker-compose up -d
+        run: |
+          ./scripts/stats.sh &
```

Comment:
It seems that running this once here is good enough. It appears to run continuously throughout the rest of the workflow.